cmd/snap/cmd_snap_op.go

```diff
@@ -113,8 +114,15 @@ type cmdInstall struct {
 }
 
 func (x *cmdInstall) Execute([]string) error {
+	var uuid string
+	var err error
+
 	cli := Client()
-	uuid, err := cli.InstallSnap(x.Positional.Snap, x.Channel)
+	if osutil.FileExists(x.Positional.Snap) {
+		uuid, err = cli.SideloadSnap(x.Positional.Snap)
```

Comment:
Don't we need to make the path absolute?  How can the daemon tell where the file lives?